src/store/zap/mutations.js

```diff
@@ -488,23 +488,22 @@ export function setDomainFilter(state, filterEnabledClusterPair) {
   let filter = filterEnabledClusterPair.filter
   state.clusterManager.filter = filter
   state.domains.map((domainName) => {
+    const openDomainValue =
+      state.clusterManager.filterString === ''
+        ? filter.domainFilterFn(domainName, state.clusterManager.openDomains, {
+            enabledClusters: filterEnabledClusterPair.enabledClusters,
+          })
+        : true
     setOpenDomain(state, {
       domainName: domainName,
-      value: filter.domainFilterFn(
-        domainName,
-        state.clusterManager.openDomains,
-        {
-          enabledClusters: filterEnabledClusterPair.enabledClusters,
-        }
-      ),
+      value: openDomainValue,
     })
   })
 }
 
 export function doActionFilter(state, filterEnabledClusterPair) {
   let filter = filterEnabledClusterPair.filter
   // When we close all, we also clear all filters.
-  resetFilters(state)
```

Comment:
a close all button should act only for closing the opened clusters , not clearing only one of the two filter options